models/intel/person-detection-0100/description/person-detection-0100.md

```diff
@@ -0,0 +1,55 @@
+# person-detection-0100
+
+## Use Case and High-Level Description
+
+This is a pedestrian detector for the Retail scenario. It is based on MobileNetV2
+backbone with two SSD heads from 1/16 and 1/8 scale feature maps and clustered
+prior boxes for 256x256 resolution.
+
+## Example
+
+![](./person-detection-0100.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP                              | 82.52%                                    |
```

Comment:
I do not see info about used dataset for AP calculation (if it is internal please specify that AP calculated on internal test set

Reply:
added